apps/sms/index.html

```diff
@@ -84,6 +85,21 @@
     <script defer src="services/js/threads.js"></script>
     <script defer src="views/inbox/js/inbox.js"></script>
     <script defer src="views/shared/js/settings.js"></script>
+    <script defer src="lib/bridge/bridge.js"></script>
+    <script defer src="lib/bridge/plugins/stream/client.js"></script>
+    <script defer src="lib/bridge/plugins/stream/service.js"></script>
+    <script defer src="lib/bridge/plugins/finalize/client.js"></script>
+    <script defer src="services/js/bridge_service_mixin.js"></script>
+    <script defer src="services/js/moz_mobile_connections/moz_mobile_connections_shim.js"></script>
+    <script defer src="services/js/moz_mobile_message/moz_mobile_message_shim.js"></script>
+    <script defer src="services/js/moz_settings/moz_settings_shim.js"></script>
+    <script defer src="services/js/moz_mobile_message/moz_mobile_message_client.js"></script>
+    <script defer src="services/js/conversation/conversation_service.js"></script>
+    <script defer src="services/js/conversation/conversation_client.js"></script>
+    <script defer src="services/js/messaging/messaging_service.js"></script>
```

Comment:
I'm not sure yet, likely not (only if we want to show something in Inbox if message is failed to send or something similar), I did it just to check if I can generalize ServiceManager to support as much services/clients as possible, but yeah there is no any reason to initialize it on startup.